script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+        except getopt.GetoptError:
+                print('buildSQLGeom.py -i <inputfile> -e <epsg> -s <site_id> -x -n <postgres_hostname> -d <dbname> -u <username> -p <password> -o <outputfile> ')
+                print('epsg default is 3857')
+                print('-o will write query to <input_filename>_SQL.txt')
+                print('-x will attempt to execute resulting query')
+                print('-n, -d, -u, -p required if -x flag enabled')
+                sys.exit(2)
+        for opt, arg in opts:
+                if opt == '-h':
+                        print('buildSQLGeom.py -i <inputfile> -e <epsg> -s <site_id> -x -n <postgres_hostname> -d <dbname> -u <username> -p <password> -o <outputfile> ')
+                        print('epsg default is 3857')
+                        print('-o will write query to <input_filename>_SQL.txt')
+                        print('-x will attempt to execute resulting query')
+                        print('-n, -d, -u, -p required if -x flag enabled')
+                        sys.exit()
+                elif opt in ("-i", "--input"):
+                        input_file = arg
+                elif opt in ("-o", "--output"):
+                        out_file = True
+                elif opt in ("-e", "--epsg"):
+                        epsg = arg
+                elif opt in ("-s", "--site"):
+                        site_id = arg
+                elif opt in ("-x", "--execute"):
+                        run_query = True
+                elif opt in ("-n", "--host"):
+                        host = arg
+                elif opt in ("-d", "--dbname"):
+                        dbname = arg
+                elif opt in ("-u", "--user"):
+                        user = arg
+                elif opt in ("-p", "--pass"):
+                        passwd = arg
+        if input_file == None:
+                print('input file is required. -h for help.')
+                sys.exit(2)
+        if out_file:
+                # Use input filename with "_SQL.txt"
+                input_ext = input_file[input_file.rfind("."):]
+                out_file = input_file.replace(input_ext, "_SQL.txt")
+
+        # Generate query by iterating through input file contents
+        query = "UPDATE sites SET geometry = ST_Geomfromtext('POLYGON(("
+
+        csv = open(input_file, 'r')
+        l = csv.readline()          # header; we can skip this
+        l = csv.readline().rstrip() # first data line
+        while l:
+                # Get values from each row of file separated by ',' and append to query
+                coords = l.split(",")
+                q_line = coords[0] + " " + coords[1] + " " + coords[2] + ","
+                query += q_line 
+                l = csv.readline().rstrip()
+
+        # Finish the end of the query and print to console
+        query += "))', "+str(epsg)+") WHERE ID="+str(site_id)
```

Comment:
For clarity I've changed var names from EPSG to be SRID, and made the default 4326. The command line argument is now -r, for (spatial) reference ID and the docs are updated accordingly.